framework/core/js/src/admin/components/PermissionGrid.tsx

```diff
@@ -366,6 +366,31 @@ export default class PermissionGrid<CustomAttrs extends IPermissionGridAttrs = I
       60
     );
 
+    items.add(
+      'hideOwnPosts',
```

Comment:
let's move this up to the participate section rather than moderation, under `allowPostEditing`.